openfisca_core/tracers.py

```diff
@@ -203,3 +218,22 @@ def print_log(self, aggregate = False):
         """
         for line in self.lines(aggregate):
             print(line)  # noqa T001
+
+
+class PerformanceLog:
+
+    def __init__(self, full_tracer):
+        self._full_tracer = full_tracer
+
+    def json(self):
+        first_tree = self._full_tracer.trees[0]
+        last_tree = self._full_tracer.trees[-1]
+        simulation_total_time = last_tree['end'] - first_tree['start']
+
+        children = [self.json_tree(tree) for tree in self._full_tracer.trees]
+        return {'name': 'simulation', 'value': simulation_total_time, 'children': children}
+
+    def json_tree(self, tree):
+        calculation_total_time = tree['end'] - tree['start']
+        children = [self.json_tree(child) for child in tree['children']]
+        return {'name': f"{tree['name']}<{tree['period']}>", 'value': calculation_total_time, 'children': children}
```

Comment:
Could we rename `value` to `calculation_time` or `time`? I initially thought it was the value of the variable.

Reply:
That's part of the interface contract with d3-flame-graph, so the name is a constraint.

Reply:
👌